TinCan/RemoteLRS.cs

```diff
@@ -28,18 +29,26 @@ limitations under the License.
 
 namespace TinCan
 {
-    public class RemoteLrs : ILrs
+    public class RemoteLrs : IRemoteLrs
     {
         public Uri Endpoint { get; set; }
         public TCAPIVersion Version { get; set; }
         public string Auth { get; set; }
         public Dictionary<string, string> Extended { get; set; } = new Dictionary<string, string>();
         public Dictionary<string, string> Headers { get; set; } = new Dictionary<string, string>();
+        private HttpClient _httpClient { get; set; }
+        //Passing FeatureFlag to TinCan to control form LaunchDarkly

```

Comment:
Remove word "LaunchDarkly" from here. I think this is a public repository, or we can remove the whole comment.